internal/bminventory/inventory.go

```diff
@@ -133,7 +133,7 @@ type InstallerInternals interface {
 	UpdateHostInstallerArgsInternal(ctx context.Context, params installer.UpdateHostInstallerArgsParams) (*models.Host, error)
 	UpdateHostIgnitionInternal(ctx context.Context, params installer.UpdateHostIgnitionParams) (*models.Host, error)
 	GetCredentialsInternal(ctx context.Context, params installer.GetCredentialsParams) (*models.Credentials, error)
-	DownloadClusterKubeconfigInternal(ctx context.Context, params installer.DownloadClusterKubeconfigParams) (io.ReadCloser, int64, error)
+	DownloadClusterKubeconfigInternal(ctx context.Context, params installer.DownloadClusterKubeconfigParams, ingress bool) (io.ReadCloser, int64, error)
```

Comment:
It's a bit strange to add boolean parameter here when it's already part of the api no? 
why not set the filename instead?

Reply:
I didn't want to open it to be able to download any file, just kubeconfig or kubeconfig-noingress

Reply:
OK i didn't see that the parameter have only cluster id. 
Will it be more explicit if the parameter will be `constants.Kubeconfig` `constants.KubeconfigNoIngress`?
You can probably define a type.

Reply:
Done